proof_transformation.ml

```diff
@@ -0,0 +1,84 @@
+open Sequent
+open Proof
+open Proof_with_notations
+open Transform_request
+
+exception Transform_exception of string;;
+
+let expand_axiom formula =
+    match formula with
+    | One -> Axiom_proof formula (* TODO apply Bottom then One ? *)
+    | Bottom -> Axiom_proof formula (* TODO apply Bottom then One ? *)
+    | Top -> Axiom_proof formula (* TODO apply Top ? *)
+    | Zero -> Axiom_proof formula (* TODO apply Top ? *)
+    | Litt _s -> Axiom_proof formula (* TODO notations *)
+    | Dual _s -> Axiom_proof formula (* TODO notations *)
```

Comment:
@olaure01 we should unfold notations if any, right?

Reply:
Yes I would say one unfolding step on each formulas.

Reply:
done